mcc/odin/odin/snapshot_list.go

```diff
@@ -13,6 +13,10 @@ func ListSnapshots(
 	result []*rds.DBSnapshot,
 	err error,
 ) {
-	result = make([]*rds.DBSnapshot, 0)
+	input := &rds.DescribeDBSnapshotsInput{}
+	output, err := svc.DescribeDBSnapshots(input)
```

Comment:
Actual query, capturing error and results, which is a `rds.DescribeDBSnapshotsOutput`, with `DBSnapshots` attribute being a `[]*rds.DBSnapshot`.